scripts/ros_release_python

```diff
@@ -30,30 +30,32 @@ def get_name_and_version():
 def release_to_pip(name, version, upload):
     _print_section('Release Python package to PIP')
 
-    cmd = ['python3', 'setup.py', 'sdist']
-    upload_targets = ['upload']
-    msgs = ['Building sdist package']
-    if upload:
-        cmd.extend(upload_targets)
-        msgs.append('upload package to PIP')
-    _print_subsection('%s...' % ', '.join(msgs))
-    if upload:
-        print('# ' + ' '.join(cmd))
-    else:
-        print('# ' + ' '.join(cmd) + '  # ' + ' '.join(upload_targets))
+    _print_subsection('Building sdist and bdist_wheel packages...')
+    cmd = ['python3', 'setup.py', 'sdist', 'bdist_wheel']
+    print('# ' + ' '.join(cmd))
     subprocess.check_call(cmd)
 
     tarball = 'dist/%s-%s.tar.gz' % (name, version)
     assert os.path.exists(tarball), "Failed to generate source tarball '%s'" % tarball
-    upload_cmd = ['scp', tarball, 'ros@ftp-osl.osuosl.org:/home/ros/data/download.ros.org/downloads/%s' % name]
+
+    upload_cmd = ['twine', 'upload', '-s', 'dist/*']
     if upload:
-        _print_subsection('Uploading package to download server...')
+        _print_subsection('Uploading package to PIP...')
     else:
-        _print_subsection('Skip uploading package to download server')
+        _print_subsection('Skip uploading package to PIP')
     print('# ' + ' '.join(upload_cmd))
     if upload:
         subprocess.check_call(upload_cmd)
 
+    scp_cmd = ['scp', tarball, 'ros@ftp-osl.osuosl.org:/home/ros/data/download.ros.org/downloads/%s' % name]
```

Comment:
We might want to drop this step of uploading to a custom location. It's now strongly recommended to host everything using the python infrastructure for security. This would be one less thing to automate and maintain.

If we do this we should remove the url from the setup.py files of the couple of packages that reference it.

Reply:
Sounds good to me. If others agree I can create PRs for them.

Reply:
Based on the offline meeting I removed the upload to downloads.ros.org in ed8efb3d2b415952a85fa167225d363e74f83ebc.